docs/hub/_sections.yml

```diff
@@ -20,7 +20,16 @@
   title: Spaces documentation (beta)
 
 - local: adding-a-library
-  title: Integrating a library to the Hub
+  title: Integrate a library with the Hub
+
+- local: how-to-downstream
+  title: How to download files from the Hub
+
+- local: how-to-upstream
+  title: How to create repositories and upload files to the Hub
+
+- local: how-to-inference
+  title: How to set up Inference API with your library
```

Comment:
to me this sounds a bit confusing vs. how to actually spawn an inference API if you're a library author

I'd suggest changing to something like `How to call the Inference API from Python code`

WDYT?